app/src/main/java/com/rai/hearthstonecard/retrofit/TokenPrefs.kt

```diff
@@ -0,0 +1,25 @@
+package com.rai.hearthstonecard.retrofit
+
+import android.content.Context
+import com.rai.hearthstonecard.R
+
+
+class TokenPrefs(context: Context) {
+
+    private val prefs = context.getSharedPreferences(context.getString(R.string.app_name), Context.MODE_PRIVATE)
+
+    fun saveAuthToken(token: String) {
+        val editor = prefs.edit()
+        editor.putString(USER_TOKEN, token)
+        editor.apply()
```

Comment:
можно использовать без объявления переменной
```suggestion
        prefs
            .edit()
            .putString(USER_TOKEN, token)
            .apply()
```